react/features/base/ui/jitsiTokens.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Let's review these and see:

- if we can drop some because they are unused
- if we can use a similar one from the base tokens